File: app/src/main/java/com/gee12/mytetroid/model/EditorAction.java
package com.gee12.mytetroid.model;

import com.gee12.htmlwysiwygeditor.ActionType;

/**
 * Типы данных для хранения параметров команд в панели инструментов редактора в базе данных.
 */
public class EditorAction {

    private ActionType type;
    private int order;
    private boolean isEnabled;
    private boolean isActive;

    public ActionType getType() {
        return type;
    }

    public int getOrder() {
        return order;
    }

    public boolean isEnabled() {
        return isEnabled;
    }

    public boolean isActive() {
        return isActive;
    }
}

File: app/src/main/java/com/gee12/mytetroid/model/EditorAction.kt
package com.gee12.mytetroid.model

import com.gee12.htmlwysiwygeditor.enums.ActionType

/**
 * Типы данных для хранения параметров команд в панели инструментов редактора в базе данных.
 */
data class EditorAction(
    val type: ActionType,
    val order: Int = 0,
    val isEnabled: Boolean = false,
    val isActive: Boolean = false,
)
